.github/workflows/check-pull-request.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: |
           mkdir -p coverage
           chmod -R a+rw ./coverage
-          docker compose -f compose.yaml -f compose.test.yaml run --rm "fcp-sfd-comms"
+          docker compose -f compose.yaml -f compose.test.yaml -f compose.override.yaml run --rm "fcp-sfd-comms"
```

Comment:
Why would you want to use the override file on a build?  That means you'll be creating networks and port bindings on the GitHub runner.

Typically the override file is for local dev needs only.  If there's bits in there that both local and CI need, then shift it to the compose.yml

Reply:
Thanks, yeah got a bit confused with my override file and my localstack start script wasnt triggered without the override. But i have moved the volume back to the base compose file.